### DISCUSSION TITLE
[Bug]: Github and Doc. buttons do not work as intended in Home Assistant, including reloading after a restart

### DISCUSSION THREAD

**Discussion by bakedhorse** [blakeblackshear/frigate#11938] [Report a Bug] [answered]
### Describe the problem you are having

The "Github" and "Documentation" buttons go to the URLs within Home Assistant Proxy/Addons.
The "Force Reload" and automatic reload after restarting frigate opens the Home Assistant's home page within the Frigate Addon/Proxy

### Steps to reproduce

1. Setup Frigate and Home Assistant, along with the proxy if needed.
2a. Click on either "Github" or "Documentation" (happens with both)
2b. Click on "Restart Frigate" and either click on "Force Reload" or wait for it to automatically refresh.
3. Presto, you are no longer on Frigate within the Frigate Addon/Proxy.

### Version

0.14.0-e56ce99

### Frigate config file

```yaml
Not relevant.
```

### Relevant log output

```shell
None.
```

### Operating system

Debian

### Install method

Docker Compose

### Network connection

Mixed

### Camera make and model

Not relevant.

### Any other information that may be helpful

_Subjective, but I like the documentation being within the addon/proxy, but it doesn't display properly_

**Comment by NickM-27** [comment, marked as answer]
Should be fixed in https://github.com/blakeblackshear/frigate/pull/11940
